### ISSUE TITLE
Asyncio compatibility

### ISSUE THREAD

**Issue by Skazu** [sighalt/logdecorator#6]
Logdecorator isn't asyncio compatible, the logs get fired before the coroutine is awaited.

See this example:
```
In [9]: @log_on_start(logging.CRITICAL, "START") 
   ...: @log_on_end(logging.CRITICAL, "END") 
   ...: async def test(): 
   ...:     print("test") 
   ...:                                                                                                                                                  

In [10]:                                                                                                                                                 

In [10]: await test()                                                                                                                                    
START
END
test
```

Expected Output was:
```
START
test
END
```

Its not even necessary to await the coroutine, see:
```
In [14]: test()                                                                                                                                          
START
END
Out[14]: <coroutine object test at 0x7f14c7cefc48>
```

**Comment by sighalt** [comment]
This feature will be available in logdecorator 2.2 and is already implemented in the release branch #7

**Comment by sighalt** [comment]
PR was merged into master branch.
